app/views/gobierto_admin/gobierto_dashboards/dashboards/list.html.erb

```diff
@@ -0,0 +1,4 @@
+<div class="modal">
+  <h2>Lorem ipsum dolor sit amet consectetur adipisicing elit. Blanditiis, illo.</h2>
+  <pre><%= params.permit!.to_json %></pre>
```

Comment:
What's doing exactly this pre block? Are you reading the params from there or it's a debug that you forgot to remove?

Reply:
Resolve this comments as they're gonna be replaced in the consequent PRs